hepdata/modules/records/utils/records_update_utils.py

```diff
@@ -183,7 +183,7 @@ def _get_url(page, specified_time, on_or_since):
     """Returns an INSPIRE API query URL for records with HEPData modified on or since a certain date."""
 
     size = RECORDS_PER_PAGE
-    url = ('https://inspirehep.net/api/literature?sort=mostrecent&size={}&page={}'.format(size, page) +
+    url = ('https://inspirehep.net/api/literature?sort=mostrecent&size={}&fields=id&page={}'.format(size, page) +
```

Comment:
This works, but [`fields`](https://github.com/inspirehep/rest-api-doc#metadata-filtering) refers to the [keys](https://inspire-schemas.readthedocs.io/en/latest/schemas/hep.html) of `hits.hits.metadata` returned by the INSPIRE-HEP search query.  The `hits.hits.id` is returned for every search and is not a key of `hits.hits.metadata`.  Since `control_number` is also returned for every value of `fields` but is a key of `hits.hits.metadata`, maybe better to replace `fields=id` with `fields=control_number`?

Reply:
Good catch! Currently it seems to work by accident that if you specify an invalid metadata field you get back the `control_number` field only. Probably shouldn't rely on that 😆